### ISSUE TITLE
Unittest Error: "ValueError: Attempted relative import in non-package"

### ISSUE THREAD

**Issue by CaitlinOCallaghan** [Netflix/vmaf#364]
Hi,

I followed the python library installation guide and came upon the following errors when running the unit test. I tried fixing the errors using past issues and postings online, but I was unable to clear them.

Thank you in advance for your help!

Terminal Error Response:

======================================================================
ERROR: feature_extractor_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: feature_extractor_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/feature_extractor_test.py", line 13, in <module>
    from .testutil import set_default_576_324_videos_for_testing, set_default_flat_1920_1080_videos_for_testing
ValueError: Attempted relative import in non-package

======================================================================
ERROR: vmafossexec_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: vmafossexec_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/vmafossexec_test.py", line 10, in <module>
    from .testutil import set_default_576_324_videos_for_testing
ValueError: Attempted relative import in non-package

======================================================================
ERROR: noref_feature_extractor_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: noref_feature_extractor_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/noref_feature_extractor_test.py", line 9, in <module>
    from .testutil import set_default_576_324_videos_for_testing
ValueError: Attempted relative import in non-package

======================================================================
ERROR: quality_runner_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: quality_runner_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/quality_runner_test.py", line 17, in <module>
    from .testutil import set_default_576_324_videos_for_testing, set_default_flat_1920_1080_videos_for_testing
ValueError: Attempted relative import in non-package

======================================================================
ERROR: result_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: result_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/result_test.py", line 16, in <module>
    from .testutil import set_default_576_324_videos_for_testing
ValueError: Attempted relative import in non-package

======================================================================
ERROR: feature_assembler_test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: feature_assembler_test
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/Cellar/python@2/2.7.16_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/caitlinocallaghan/vmaf/python/test/feature_assembler_test.py", line 12, in <module>
    from .testutil import set_default_576_324_videos_for_testing
ValueError: Attempted relative import in non-package

----------------------------------------------------------------------
Ran 195 tests in 86.354s

FAILED (errors=6)

**Comment by zsimic** [comment]
That is an unfortunate `unittest` quirk, not sure exactly what's causing it, but `unittest` incorrectly fails to see the `test/__init__.py` file somehow.

You do have the option of using `pytest` instead, in your activated venv, run this:
```
pytest test/feature_assembler_test.py
```

That will execute the test (via pytest https://docs.pytest.org/en/latest/), which is quite better at all those details.

Another thing you can do is also add `-t .` (just found this out) to you `unittest` invocation, ie:
```
python -m unittest discover -t . -s test/ -p 'feature_assembler_test.py'
```

**Comment by li-zhi** [comment]
thanks for the pointer!

**Comment by li-zhi** [comment]
Fixed by #368. Let me know if things work out for you @CaitlinOCallaghan
